user_input.py

```diff
@@ -0,0 +1,59 @@
+from textwrap import dedent
+from time import sleep
+from random import choice
+
+savage1 = {"ducky": "Has feathers and goes quack quack", "red": "The colour your face should be", "cup": "sippy sippy", "feeble": "An assessment of your mental prowess..."}
+
+
+def check_input(user_input):
```

Comment:
This function has a return signature of `None | True` - you probably want `Boolean` aka `True | False`.

Reply:
```suggestion
def check_input(user_input: string) -> bool:
```

I think